README.md

```diff
@@ -129,25 +129,60 @@ mqttui "topic"
 
 ## Install
 
-### Packaged
-
 [![Packaging status](https://repology.org/badge/vertical-allrepos/mqttui.svg?columns=2&exclude_unsupported=1)](https://repology.org/project/mqttui/versions)
 
-#### Arch Linux
+Check the [Releases](https://github.com/EdJoPaTo/mqttui/releases).
+
+There are binary releases for Windows, Debian/Ubuntu, RPM based distros, and tarball archives.
+There are binaries for different CPU architectures:
+
+- Intel/AMD - x86-64
+- 64 bit ARMv8 - aarch64
+- 64 bit RISC-V - riscv64gc
+- 32 bit ARMv7 - armv7
+- 32 bit ARMv6 - arm
+
+### Homebrew (Mac or Linux)
 
 ```bash
-pacman -S mqttui
+brew install mqttui
 ```
 
-#### Homebrew (Mac or Linux)
+### Linux package managers
+
+#### Arch Linux
 
 ```bash
-brew install mqttui
+pacman -S mqttui
 ```
 
-### Prebuilt
+#### Debian, Ubuntu and Deb based Linux
 
-Check the [Releases](https://github.com/EdJoPaTo/mqttui/releases).
+Download the appropriate .deb for your architecture and run:
+
+`sudo dpkg -i <downloaded file>`
+
+#### Redhat, Centos and RPM based Linux
+
+Download the appropriate .rpm for your architecture and run:
+
+`sudo rpm -i <downloaded file>`
+
+### Tarball for other Linuxes and MacOS
+
+The binaries are also shipped as plain tarballs, targetting each architecture for any generic Linux or macOS.
+
+Note that the linux binaries are built for glibc based systems. You will need to compile [from source](#from-source) for other libc/musl systems like Alpine Linux.
+
+You need to extract the tarball and put the binary somewhere on your path (common locations are `~/bin`, `~/.local/bin` or `/usr/local/bin`).
+
+### Windows
+
+From the latest release, select `mqttui-<version>-<arch>-pc-windows-msvc.zip`
```

Comment:
Not sure about this sentence. Like, everything here is from the release page, it feels a bit superfluous. I think I would just remove this one as arch is already explained above and windows is relatively clear. Also arch is mentioned in the next sentence too. Or do I miss something?